CHANGELOG.md

```diff
@@ -28,6 +28,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed `lt_dev_stm32_nucleo_f439zi` to `lt_dev_stm32_nucleo_f439zi_t`.
 - Added sections **Supported Host Platforms** and **Supported Cryptographic Functionality Providers** into **Other** section in the Libtropic documentation.
 - Added dependencies on [micro-ecc](https://github.com/kmackay/micro-ecc) and [ed25519](https://github.com/orlp/ed25519/tree/master) repositories in functional tests (for verification of signatures calculated by TROPIC01).
+- Updated L3 result values according to Application FW 1.1.0:
+  - Renaming inside `lt_l3_process.h` (not exposed to users directly).
+  - Renaming of `lt_ret_t` values:
+    - `LT_L3_PAIRING_KEY_EMPTY` to `LT_L3_SLOT_EMPTY`.
+    - `LT_L3_PAIRING_KEY_INVALID` to `LT_L3_SLOT_INVALID`.
+    - `LT_L3_ECC_INVALID_KEY` to `LT_L3_INVALID_KEY`.
+    - `LT_L3_R_MEM_DATA_WRITE_WRITE_FAIL` to `LT_L3_SLOT_NOT_EMPTY`.
+    - `LT_L3_R_MEM_DATA_WRITE_SLOT_EXPIRED` to `LT_L3_SLOT_EXPIRED`.
+    - `LT_L3_MCOUNTER_UPDATE_UPDATE_ERR` to `LT_L3_UPDATE_ERR`.
+- Reordered values inside `lt_ret_t` enum.
```

Comment:
As the users shouldn't rely on the specific value, order does not matter for them.
```suggestion
```